BREEZE.rst

```diff
@@ -2150,6 +2155,11 @@ This is the current syntax for  `./breeze <./breeze>`_:
   --no-rbac-ui
           Disables RBAC UI when Airflow 1.10.* is installed.
 
+  --install-wheels
```

Comment:
What is the reason to have duplicated information?

Reply:
It's automated output from --help-all in breeze - and this flag is valid for two command - enter breeze and start-airflow